src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AutoInsert/OnAutoInsertEndpoint.cs

```diff
@@ -64,7 +64,7 @@ public async Task<OnAutoInsertResponse> Handle(OnAutoInsertParams request, Cance
         {
             var document = await Task.Factory.StartNew(() =>
             {
-                _documentResolver.TryResolveDocument(request.TextDocument.Uri.GetAbsoluteOrUNCPath(), out var documentSnapshot);
+                _documentResolver.TryResolveDocument(request.TextDocument.Uri.ToUri().GetAbsoluteOrUNCPath(), out var documentSnapshot);
```

Comment:
Instead of doing `ToUri` everywhere why not just add a GetAbsoluteOrUNCPath extension method that operates off of `TextDocumentUri`?